cloud/azure/initial_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+RESOURCE_GROUP_NAME=tfstate
+STORAGE_ACCOUNT_NAME=tfstate$RANDOM
+CONTAINER_NAME=tfstate
+
+echo "Create resource group"
+az group create --name $RESOURCE_GROUP_NAME --location eastus
+
+echo "Create storage account"
+az storage account create --resource-group $RESOURCE_GROUP_NAME --name $STORAGE_ACCOUNT_NAME --sku Standard_LRS --encryption-services blob
+
+echo "Create blob container"
+az storage container create --name $CONTAINER_NAME --account-name $STORAGE_ACCOUNT_NAME
```

Comment:
question (if-minor): how feasible would it be to make this code idempotent?

Reply:
yeahhh was thinking about this itsss maybe a lil tricky due to the needs to be globally unique aspect. basically i don't think it's totally worth the effort but i added some extra docs. maybe could use something like git branch or date or some other like less random but still unique thing here instead which wouldn't be idempotent but would be maybe less spooky

Reply:
okay yes shubha and i chatted about this in our sync and am gonna use the like if it exists then do x

Reply:
great!